config/csm/kustomization.yaml

```diff
@@ -0,0 +1,37 @@
+# Use DP0 (real hardware) for the base.
+bases:
+- ../dp0
```

Comment:
Should we rename this to `hardware` to be clearer?

Reply:
We could.  I'll let that be a separate PR.  That'll require an update to systems.yaml in nnf-deploy.